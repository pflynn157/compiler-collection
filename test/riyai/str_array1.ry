
func main -> i32 is
    array s : string[5];
    print(s);
    
    s[0] := "Hi";
    s[1] := "How are you?";
    s[2] := "I am good.";
    s[3] := "And you?";
    s[4] := "Yeah";
    
    print(s);
    
    var i : i32 := 0;
    while i < 5 do
        print(s[i]);
        i := i + 1;
    end
    
    return 0;
end

