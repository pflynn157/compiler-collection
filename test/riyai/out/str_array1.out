["", "", "", "", ""]
["Hi", "How are you?", "I am good.", "And you?", "Yeah"]
Hi
How are you?
I am good.
And you?
Yeah
